lib/msf/ui/console/driver.rb

```diff
@@ -724,6 +724,46 @@ def choose_readline(opts)
 
     begin
       require 'readline'
+
+      # Only Windows requires a monkey-patched RbReadline
+      return unless Rex::Compat.is_windows
+
+      if defined?(::RbReadline)
```

Comment:
```suggestion
      if defined?(::RbReadline) && !defined?(RbReadline::refresh_console_handle)
```

I don't think we need to monkey patch this multiple times